NEWS.md

```diff
@@ -4,6 +4,40 @@ PgBouncer changelog
 PgBouncer 1.22.x
 ----------------
 
+**2024-06-28  -  PgBouncer 1.22.2  -  "Into the new beginnings"**
+- Features
+  * Add support for user name maps for for `cert` and `peer` authentication
+    methods. ([#996])
```

Comment:
I think you can expand a little bit more on this (this is one of the major features of the release). Something like: This allows a lot more freedom for client-side TLS certificate authentication.